ansible/README.md

```diff
@@ -3,20 +3,21 @@ This will ease installation, and deployment of the pbench dashboard.
 
 ## Required
 - Ansible needs to be installed on the host where you want to run this playbook
-- An inventory file containing the following key values defined:
-  - "`elasticsearch_url`", "`results_url`", "`graphql_url`", "`run_index`", "`prefix`"
-    See the `/README.md` for more details. 
+- An inventory file containing the server values defined
+
+## Endpoint Configuration
+API endpoints are configured and bundled into the final binary when running `yarn build`. Before deployment of the binary to a remote host, consider the current configuration definition at `config/endpoints.js`.
```

Comment:
Interesting, since I was playing with this yesterday afternoon trying to deploy a dashboard to my VM. I'd forgotten you'd merged the `process.env` change and initially built and deployed "the old way", which didn't work.

Then when I remembered and edited `endpoints.js` I found that `yarn build` didn't include the changes ... I had to delete the `dist/` directory contents and rebuild from scratch. Maybe worth a note that apparently just changing `endpoints.js` and `yarn build` isn't enough to pick up the changes?

Reply:
Running `yarn build` should overwrite the contents in `dist/` according to the UmiJS docs (`yarn build` maps to `umi build` in `package.json`). I was able to deploy the latest dashboard changes to staging without having to delete the `dist/` directory. Perhaps clearing the yarn cache might help here? https://classic.yarnpkg.com/en/docs/cli/cache/